### PR TITLE
feat: section analytics

### PR DESCRIPTION
**Pull request by tuncaulubilge**
Analytics for section pages: https://nidigitalsolutions.jira.com/browse/REPLAT-4940

Events:
- Page load event on section page
- OnPress event on tiles. Added tracking events to each tile and added `tileName` to each tile within the slice.
- Scroll depth for slices.

PageView:
```
action: "Viewed"
attrs: {
  eventTime: "2019-02-21T17:35:02.142Z"
  sectionName: "News"
}
component: "Page"
object: "Section"
```

Scroll:
```
action: "Scrolled"
attrs: Object
  eventTime: "2019-02-21T17:36:01.125Z"
  scrollDepth: Object
    itemNumber: 3
    name: "LeadOneAndOneSlice"
    total: 9
  sectionName: "News"
component: "SectionChild"
object: "Section"
```

Press:
```
action: "Pressed"
attrs: Object
  articleHeadline: "Driving Off"
  articleId: "522abf2c-e752-11e8-9f32-f2801f1b9fd1"
  articleUrl: "/article/123"
  eventTime: "2019-02-21T17:36:41.840Z"
  sectionName: "News"
  sliceDepth: Object
    itemNumber: 5
    total: 9
  sliceName: "LeadTwoNoPicAndTwoSlice"
  tileName: "lead1"
component: "Tile"
object: "Slice"
```

### CODE REVIEW

File: packages/fixture-generator/src/mock-section.ts [hunk 1]
```diff
@@ -33,7 +34,15 @@ function getSlices(): Array<ArticleSlice> {
 }
 
 function getPuzzleSlices(): Array<PuzzleSectionSlice> {
-  return [];
```
Comment (jimhunty) [inline packages/fixture-generator/src/mock-section.ts]: Do tests need updating?

File: packages/pages/src/section/index.js [hunk 2]
```diff
@@ -6,10 +6,15 @@ import Section from "@times-components/section";
 import withNativeProvider from "../with-native-provider";
 
 const onPress = () => {};
+const track = () => {};
```
Comment (jimhunty) [inline packages/pages/src/section/index.js]: Assume this and onPress are empty functions as they require the Android implementation?

File: packages/fixture-generator/src/mock-section.ts (hunk 1 above)
Comment (tuncaulubilge) [reply packages/fixture-generator/src/mock-section.ts]: Will do :+1:

File: packages/pages/src/section/index.js (hunk 2 above)
Comment (tuncaulubilge) [reply packages/pages/src/section/index.js]: Yepp, these will be using NativeModules once we add a SectionEvents module into the native apps

**Pull request merged**